hack/istio/tempo/install-tempo-env.sh

```diff
@@ -45,6 +46,10 @@ while [[ $# -gt 0 ]]; do
       INSTALL_KIALI="$2"
       shift;shift
       ;;
+    -sc|--secure-distributor)
```

Comment:
trivial, but... the abbreviation should be the first letter of the two words;
```suggestion
    -sd|--secure-distributor)
```

Reply:
Changed, I'm not sure where I get the c from :)